src/services/print.js

```diff
@@ -364,8 +364,9 @@ ngeo.Print.prototype.encodeVectorLayer_ = function(arr, layer, resolution) {
         var style = styles[j];
         var styleId = goog.getUid(style).toString();
         var featureStyleProp = ngeo.Print.FEAT_STYLE_PROP_PREFIX_ + j;
+        var geometryType = feature.getGeometry().getType();
```

Comment:
You just need to call `getGeometry().getType()` once for each feature. And you also want to deal with the case where the feature does not have a geometry (`getGeometry` returns `null` or `undefined`).